src/script/squirrel_helper.hpp

```diff
@@ -44,6 +44,14 @@ namespace SQConvert {
 	template <> struct Return<void *>       { static inline int Set(HSQUIRRELVM vm, void *res)       { sq_pushuserpointer(vm, res); return 1; } };
 	template <> struct Return<HSQOBJECT>    { static inline int Set(HSQUIRRELVM vm, HSQOBJECT res)   { sq_pushobject(vm, res); return 1; } };
 
+	template <typename T> requires std::is_enum_v<T> struct Return<T> {
+		static inline int Set(HSQUIRRELVM vm, T res)
+		{
+			sq_pushinteger(vm, static_cast<int64_t>(res));
```

Comment:
Is `to_underlying(res)` workable here, or does it need to be `int64_t`?

Reply:
No, that seems to work as well.